includes/template-tags.php

```diff
@@ -61,3 +61,67 @@ function adjust_brightness( $hex, $steps ) {
 	return $return;
 
 }
+
+/**
+ * An svg template helper.
+ *
+ * @param  string  $name   The SVG Name.
+ * @param  array   $opts   Optional. The SVG options such as Role, Class, Label and Hidden.
+ * @param  boolean $output Either return or output the SVG Icon string.
+ * @return html
+ */
+function svg_icon( $name = '', $opts = array(), $output = false ) {
+	if ( empty( $name ) ) {
+		return '';
+	}
+
+	if ( is_customize_preview() ) {
```

Comment:
the $name in the generated sprite file is prefacing the name with `sprite-` can we add that in here so in the function call you don't have to call sprite-[name]?

That way we can just do `<?php echo TenUpScaffold\Utility\svg_icon( 'home', ['height' => '75', 'width' => '75'] ); ?>` for the function call